taotie/sources/arxiv.py

```diff
@@ -20,7 +21,9 @@ class Arxiv(BaseSource):
 
     def __init__(self, sink: MessageQueue, verbose: bool = False, **kwargs):
         BaseSource.__init__(self, sink=sink, verbose=verbose, **kwargs)
-        self.authors = os.environ.get("ARXIV_AUTHORS", "").split(",")
+        with open('arxiv_author.json') as f:
+            author_dict = json.load(f)
+        self.authors = author_dict
```

Comment:
Please check the format of the data you load from the file. It is a dict, where each value is a list of authors. You may process them correctly.